shuffle/cli/api.md

```diff
@@ -0,0 +1,143 @@
+# Shuffle API
+
+All commands are shown like so: `generic_command` | `example_command`
+
+## shuffle new
+
+**Usage:**
+
+`shuffle new <project_path>` | `shuffle new ~/Desktop/TestCoin`
+
+Takes in a project_path and creates a new shuffle project at that location with all pre-generated folders which can be used down the line. Also creates ~/.shuffle/Networks.toml which is your networks configuration file.
+
+Note: Feel free to add any network to connect to with the same format as the localhost network (already in the file) to Networks.toml.
+
+## shuffle node
+
+**Usage:**
+
+`shuffle node`
+
+Runs a local node at endpoint http://127.0.0.1:8080 which listens to all transactions sent by the user. Also creates ~/.shuffle/nodeconfig directory which contains information to configure the local node.
+
+`shuffle node --genesis <move_package_path>` | `shuffle node --genesis diem-move/diem-framework/experimental`
+
+Runs a local node with specific move package as the genesis modules.
+
+## shuffle account
+
+**Usage:**
+
+`shuffle account`
+
+Creates an account on localhost by claiming an address. Also saves account information locally in the user's ~/.shuffle/networks/localhost/accounts.
```

Comment:
Does this tie into the on-chain accounts? How does this tie back into requirements for genesis?

Reply:
1. This does creates accounts on-chain. 
2. What does your second question mean?